scripts/dev_cli.sh

```diff
@@ -171,8 +171,9 @@ cmd_help() {
     echo "    clean [<cargo args>]]"
     echo "        Remove the Cloud Hypervisor artifacts."
     echo ""
-    echo "    shell"
+    echo "    shell [--libc musl|gnu]"
```

Comment:
Why do we need this option? The way the `shell` command works, the user has access to the container itself, which means he can decide whether to use `gnu` or `musl`, therefore there's no need for this extra parameter.
Or am I missing something here?

Reply:
I was using it mainly to run `integration` test manually (after setup something in the container). For example: 1) start the container shell w/ `scripts/cli_dev.sh shell --libc musl` and cleared the `dmesg/syslog` inside the container, 2) run `integration tests` manually w/ `scripts/run_integration_tests_x86_64.sh/` in side the container, 3) start another container shell w/ `docker exec` to collect the dmesg/syslog. 

I am not sure how common this use-case is and there may be alternative way to do it w/o the `shell` command. I agree that enabling the `libc` flag for `shell` command can be misleading, as users can decide whether to build w/ gnu or musl after all. 

After all, I feel we will need to adjust the `shell` command in the script, as we are using the `${libc}` value that is never set. But we may just need to remove it (`--env CH_LIBC="${libc}"`) while launching the container shell.  WDYT?

Reply:
Ah yes I didn't see that `--env CH_LIBC="${libc}"` was part of the docker command to create the container.
IMO, we should simply remove it, because we just want to give the user access to a container where he can do whatever he wants, so let's not add some options like `libc`.
The way you would do what you described, you would start the container and then you would run the integration tests script with `CH_LIBC=musl` or `CH_LIBC=gnu`.